pkg/core/interop/runtime/util.go

```diff
@@ -74,3 +77,19 @@ func GetNetwork(ic *interop.Context) error {
 	ic.VM.Estack().PushVal(uint32(m))
 	return nil
 }
+
+// GetRandom returns pseudo-random number which depends on block nonce and transaction hash.
+func GetRandom(ic *interop.Context) error {
+	res := murmur128(ic.NonceData[:], ic.Network)
+	ic.VM.Estack().PushVal(bigint.FromBytesUnsigned(res))
+	copy(ic.NonceData[:], res)
+	return nil
+}
+
+func murmur128(data []byte, seed uint32) []byte {
+	h1, h2 := murmur3.SeedSum128(uint64(seed), uint64(seed), data)
+	result := make([]byte, 16)
```

Comment:
Frankly, we might as well do `make([]byte, 17)` here prepending `00` to Murmur output and avoid `bigint.FromBytesUnsigned` (I'm not sure it's useful otherwise). At the same time `bigint.FromBytesUnsigned` is a bit more optimized, so I'm not against it.

Reply:
I was thinking about making this local to the package. Decided to place in `bigint` because of the tests.